### ISSUE TITLE
Jobs table always empty

### ISSUE THREAD

**Issue by ishwon** [CachetHQ/Cachet#2740]
Hello,

My config contains the following:

```
CACHE_DRIVER=apc
SESSION_DRIVER=apc
QUEUE_DRIVER=database
```

When adding a subscriber no verification email is being sent. The `jobs` table is empty. I have the following command as cronjob:

```
php /var/www/cachet/artisan schedule:run >> /dev/null 2>&1
```

No errors show up in the laravel log. I am on the `2.4` branch.

**Comment by GrahamCampbell** [comment]
Please confirm what is the name of that env file in full, and what is the output when you type in `app('config')['queue.default']` into `php artisan tinker` in the cachet main folder please?

**Comment by ishwon** [comment]
The file name is `.env` and running `php artisan tinker` from `/var/www/cachet` produces the following from:

```                                                              
  [Symfony\Component\Console\Exception\CommandNotFoundException]  
  Command "tinker" is not defined.                                
```

**Comment by jbrooksuk** [comment]
Tinker is now available again in 2.4

If you're using the queue database, you need to not only run `schedule:run` but also `queue:work`. https://cachet.readme.io/docs/configuring-the-queue#section-installations-from-cachet-v24-dev-onwards
